packages/ui/src/index.html

```diff
@@ -5,6 +5,8 @@
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover">
   <title>Logto</title>
+  <link rel="preload" href="/api/.well-known/sign-in-exp" as="fetch" crossorigin="anonymous">
+  <link rel="preload" href="/api/phrase" as="fetch" crossorigin="anonymous">
```

Comment:
It seems Safari doesn't support this
![Uploading image.png…]()

Reply:
doesn't hurt if safari silently ignores?

Reply:
thanks actually Safari even has a better dev experience lol

<img width="1126" alt="image" src="https://user-images.githubusercontent.com/14722250/224282302-f2971e90-6309-4b9c-8e4d-4c8c423f7870.png">

Reply:
oh sorry Chrome has this warning when needed as well jaja

Reply:
https://stackoverflow.com/questions/41655955/why-are-preload-link-not-working-for-json-requests
https://stackoverflow.com/questions/52635660/can-link-rel-preload-be-made-to-work-with-fetch

i think there are a lot of issues in Safari. should we keep this since Chrome works?

Reply:
Seems it doesn't break anything. OK, then